doc/source/how-to/deprecating.rst

```diff
@@ -0,0 +1,126 @@
+   are available, and any relevant timelines. See a template below for the issue content.
+
+   .. warning::
+
+      Make sure to adapt the template to your specific library and situation.
+
+   .. code:: markdown
+
+      ## ⚠️ Project Deprecation Notice
+
+      **This repository is no longer maintained as of [DATE].**
+
+      ### Reason for deprecation
+      [Explain briefly why the project is being deprecated — e.g. better alternatives, no time to maintain, outdated use case.]
+
+      ### Alternatives
+      If you're looking for a maintained alternative, consider:
+      - [Alternative 1](https://...)
+      - [Alternative 2](https://...)
+
+      ### What this means
+      - No further updates, bug fixes, or pull request reviews
+      - Issues will be closed
+      - The repository will be archived
+
+      Thank you to everyone who contributed, used, or supported this project!
+
+   This issue serves as a permanent record of the deprecation and provide users with
+   necessary information about alternatives.
+
+   Make sure to pin the issue to the top of the repository so that it is easily visible to users.
+   This can be done by selecting the "Pin issue" option in the issue's right-side menu, on GitHub.
+
+   See an example at `PyAdditive Widgets deprecation issue`_.
+
+4. **Adapt the README**: Update the library's ``README`` file to reflect the deprecation.
+   This should include a clear notice at the top of the ``README``, informing users that the
+   library is deprecated. See an example at `PyAdditive Widgets README`_.
+
+5. **(Optional) Add a warning in the code**: If applicable, add a warning in the code itself to inform users
+   that the library is deprecated. This can be done using Python's `warnings` module. For example:
+   
+   .. note::
+
+      Make sure to adapt the URL in the warning message.
+
+   .. code:: python
+
+      # At the top of your main module or package (i.e. src/ansys/<...>/__init__.py)
+
+      import warnings
+
+      warnings.warn(
+          "This library is deprecated and will no longer be maintained. "
+          "Please consider using alternatives. "
+          "For more information check https://github.com/ansys/<repository>/issues/<number>",
+          DeprecationWarning,
+      )
+
+   See an example at `PyAdditive Widgets deprecation warning`_.
+
+6. **(Optional) Make a last release**: If you carried out step 5, consider making a final release
+   of the library that includes the deprecation warning. This ensures that users who install
+   the library in the future see the warning immediately.
+
+7. **Archive the repository**: Once the deprecation issue is created and the ``README`` is updated,
+   you can archive the repository. This prevents any further changes to the repository and
+   signals to users that the library is no longer maintained. To archive a repository, go to the
+   repository settings and select "Archive this repository."
```

Comment:
Is it possible to revert this action? If it's not the case, we should add a warning.

Reply:
Yes you can revert it